less/_defaults/_spacing-containers.less

```diff
@@ -36,11 +36,12 @@
 // Block padding
 // --------------------------------------------------
 @block-padding-ver: 4rem;
-@block-padding-hoz: .5rem;
+@block-padding-hoz: 1rem;
 
 // Component padding
 // --------------------------------------------------
-@component-padding-hoz: .5rem;
+@component-padding-ver: 1rem;
```

Comment:
@guywillis, any particular reason for increasing the padding to 1rem? I'd be inclined to keep this as .5rem as it aligns the content with the header for consistency. Also I think we purposely reduced the content padding in v5 to allow more room for content (1rem in total between left/right components on desktop, .5rem horizontal padding on mobile).

Reply:
@kirsty-hames Increased the padding as, when combined with either `bg-color` or `comp-header-bg-color`, 1rem looked nicer.

I think we could update the other existing .5rem to 1rem so they're all in alignment - observationally, 8px gap between the edge of the content and the edge of the screen is pretty small and increasing this width, in my view, makes the small screen display more appealing. 

Ultimately, I think we should look to increase the content display area for larger screens which should negate the increase in padding.

Reply:
I agree that `comp-header-bg-color` looks better with the increase of padding but seeing as this is an optional style I think the padding should be specific to this style and not global. 
I know it's personal preference but I've never had a request to increase the horizontal padding on projects. For mobile in particular I find more space for content is preferred.

Reply:
I would agree but I fear the potential mismatch with some components with .5 padding and some set at 1rem depending on whether they have a custom class applied or not - the idea behind the change is to ensure consistency at the expense of a few pixel padding.